#!/usr/bin/env python
PACKAGE="navigation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen=ParameterGenerator()

gen.add("start_cmd", bool_t, 0, "Start/Stop", False)
gen.add("path_from_file", bool_t, 0, "Read/Not", False)
gen.add("initial_turning_side",    str_t,    0, "Initial turning side",  "left")

gen.add("corridor_width",        double_t, 0, "corridor_width: ",          2.0,     0.5, 4.0)
gen.add("row_dim",        double_t, 0, "row_dim: ",          0.2,     0.1, 2.0)
gen.add("row_length",        double_t, 0, "row_length: ",          2,     0.1, 200.0)

exit(gen.generate(PACKAGE, "vuoto", "navigation_controller"))
